notebooks/03_evaluate/evaluation.ipynb

```diff
@@ -48,8 +48,8 @@
      "name": "stdout",
```

Comment:
<p>maybe here we can add that Log loss penalize heavily classifiers that are confident about incorrect classifications.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/591/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>